pykickstart/commands/autopart.py

```diff
@@ -526,7 +526,7 @@ def _getParser(self):
                         Do not create a /boot partition.""")
         op.add_argument("--noswap", action="store_true", default=False,
                         version=F26, help="""
-                        Do not create a swap partition.""")
+                        Do not create a swap partition. Incompatible with --hibernation.""")
```

Comment:
Looking at other examples of exclusive options it looks like we have historically used the same wording as the runtime error, added to the help. So this should be similar to autopart's --pbkdf-time help string -- `Only one of ``--noswap`` and ``--hibernation`` can be specified.` Note the backtick usage here, they are used by sphinx when generating the docs.